src/components/atoms/ScrollToTopButton/index.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { fireEvent, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { testRenderer } from "../../../setupTests";
+import ScrollToTopButton from "./index";
+
+describe("<ScrollToTopButtom />", () => {
```

Comment:
BTW around this topic of testing our new component, jest has its limitations (what we already know from other tests that can't be written here): https://github.com/testing-library/react-testing-library/issues/671
We may think of adding puppeteer or Cypress as KCD suggests 🙃

Reply:
I saw this discussion earlier today (when i was googling how to test scrolling) and i want to talk about KCD suggestions